Lighthouse Camera/UI/Main/ViewController.swift

```diff
@@ -111,6 +110,9 @@ class ViewController: UIViewController {
     self.isBusy = false;
     sViewController = self;
 
+    // Initialize the delegate to self indicating viewcontroller is going to implemenent PreviewViewDelegate
+    self.previewView.delegate = self
```

Comment:
Mmmh... Creating cycles is not a very good idea in Swift, as there is no garbage-collection. This is probably not going to hurt us at runtime, since the view is never going to disappear, but I have the intuition that it might be problematic in some tests.

If you do this, can you break the cycle somewhere (and document that you have broken it)?